README.md

```diff
@@ -2,6 +2,11 @@
 
 > A consistent store for your hypercore feeds.
 
+[![Build Status](https://travis-ci.com/dxos/feed-store.svg?branch=tinchoz49-style)](https://travis-ci.com/dxos/feed-store)
+[![Coverage Status](https://coveralls.io/repos/github/dxos/feed-store/badge.svg?branch=tinchoz49-style)](https://coveralls.io/github/dxos/feed-store?branch=tinchoz49-style)
+[![js-semistandard-style](https://img.shields.io/badge/code%20style-semistandard-brightgreen.svg?style=flat-square)](https://github.com/standard/semistandard)
```

Comment:
This is going to be really painful but it's the right decision.

I'm just sad that so many people on the internet are wrong.

Reply:
at least we are going to have something in common with these companies: https://standardjs.com/#who-uses-javascript-standard-style